### PR TITLE
Add static pages

### PR DESCRIPTION
**Pull request by halo93**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <li class="<%= active_class help_path %>"><a href="<%= help_path %>">
+          <i class="fa fa-heartbeat fa-lg"></i> <%= t "help" %></a>
+        </li>
+        <li class="<%= active_class about_path %>"><a href="<%= about_path %>">
+          <i class="fa fa-group fa-lg"></i> <%= t "about" %></a>
+        </li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#"><span class="fa fa-user fa-lg"></span> Sign Up</a></li>
+        <li><a href="#"><span class="fa fa-sign-in fa-lg"></span> Login</a></li>
```
Comment (phanbt58) [inline app/views/layouts/_header.html.erb]: I18n

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "title"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```
Comment (nhunght) [inline app/helpers/application_helper.rb]: page_title.empty? ? base_title : page_title + " | " + base_title

File: app/helpers/static_pages_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (nhunght) [inline app/helpers/static_pages_helper.rb]: remove file

File: app/views/layouts/_head.html.erb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<%= csrf_meta_tags %>
+<meta name="viewport" content="width=device-width, initial-scale=0.7">
+<%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
```
Comment (nhunght) [inline app/views/layouts/_head.html.erb]: media: :all

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,12 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= t ".title" %></title>
-    <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <title><%= full_title yield(:title) %></title>
```
Comment (nhunght) [inline app/views/layouts/application.html.erb]: remove ()

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,12 +1,16 @@
...
+    <%= render "layouts/head" %>
+    <%= render "layouts/shim" %>
   </head>
   <body>
+  <%= render "layouts/header" %>
+  <div class="container" id="main-content">
     <%= yield %>
+    <%= render "layouts/footer" %>
+  </div>
+  <%= debug(params) if Rails.env.development? %>
```
Comment (nhunght) [inline app/views/layouts/application.html.erb]: remove this line

**Pull request merged**